shuup/front/api/orders.py

```diff
@@ -0,0 +1,104 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2017, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import mixins, serializers, viewsets
+from shuup.api.mixins import PermissionHelperMixin
+from shuup.core.api.address import AddressSerializer
+from shuup.core.api.orders import OrderFilter, OrderStatusChangeMixin
+from shuup.core.models import get_person_contact, Order, OrderLine, Shop
+
+
+class ShopSerializer(serializers.ModelSerializer):
+    logo = serializers.SerializerMethodField()
+    contact_address = AddressSerializer()
+
+    class Meta:
+        model = Shop
+        fields = ("id", "name", "logo", "options", "contact_address")
+
+    def get_logo(self, shop):
+        if shop.logo:
+            return self.context["request"].build_absolute_uri(shop.logo.url)
+
+
+class OrderLineSerializer(serializers.ModelSerializer):
+    image = serializers.SerializerMethodField()
+    taxful_price = serializers.SerializerMethodField()
+
+    class Meta:
+        model = OrderLine
+        fields = "__all__"
+
+    def get_image(self, line):
```

Comment:
just a comment: we need to put this on a Field as other common tasks we do.. we are repeating ourselves 😄 
I will put this on my TODO list to be done on the API refactor.